aries-site/src/pages/guidelines/philosophy.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { ContentSection, PageLayout, Subsection } from '../../layouts';
+import { SubsectionText } from '../../components';
+
+const title = 'Philosophy';
+
+const Philosophy = () => (
+  <PageLayout title={title}>
+    <ContentSection>
+      <Subsection level={1} name={title} topic="Guidelines">
```

Comment:
Should we create a `const topic = 'Guidelines'` on line 6 similar to how `title` is treated?

Reply:
Yes I think that would be a good idea.